app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "1.2.14",
+  "version": "1.2.15",
```

Comment:
Is this the last PR for this ticket? Its probably 1.3 once the ticket is done right?